clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -223,14 +238,47 @@ const TcfVendors = ({
         const dataCategories: GvlDataCategories | undefined =
           // @ts-ignore the IAB-TCF lib doesn't support GVL v3 types yet
           gvl?.dataCategories;
+        const isConsent =
+          vendorRecordsWithLegalBasis(
+            [vendor],
+            LegalBasisForProcessingEnum.CONSENT
+          ).length === 1;
+        const isLegint =
+          vendorRecordsWithLegalBasis(
+            [vendor],
+            LegalBasisForProcessingEnum.LEGITIMATE_INTERESTS
+          ).length === 1;
         return (
           <DataUseToggle
-            dataUse={{ key: vendor.id, name: vendor.name }}
+            dataUse={{
+              key: `${vendor.id}-legint`,
+              name: vendor.name,
+            }}
             onToggle={() => {
-              handleToggle(vendor);
+              handleToggle(
+                vendor,
+                LegalBasisForProcessingEnum.LEGITIMATE_INTERESTS
+              );
             }}
-            checked={enabledIds.indexOf(vendor.id) !== -1}
+            checked={enabledVendorLegintIds.indexOf(vendor.id) !== -1}
             badge={gvlVendor ? "IAB TCF" : undefined}
+            secondToggle={
+              <div
+                style={{ width: "50px", display: "flex", marginLeft: ".2em" }}
```

Comment:
because we aren't using a table, we force positioning this way 😬 follow up ticket to refactor to make things into proper tables: https://github.com/ethyca/fides/issues/4190